### ISSUE TITLE
Swagger support

### ISSUE THREAD

**Issue by gperdomor** [mulesoft/api-workbench#504]
Hi... Since Mulesoft joined to OpenAPI Initiative, we can expect Swagger support in Api Workbench

**Comment by sichvoge** [comment]
I don't think we will add any support for Swagger (now called OpenAPI) anytime soon. The API Workbench still is a fully fledged IDE plugin built around the RAML modeling language.

That does not mean we completely reject your request though, but we have other priorities at the moment like moving our core functions into the RAML language server.
